starter_code/models/Movies.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require("mongoose")
+const Schema = mongoose.Schema
+
+const movieSchema = new Schema({
+    title : String,
+    director: String,
+    stars: Array,
```

Comment:
to guarantee the similarity of your data, you can precise that stars is an array that containing string:
stars: [String]